pkg/utils/utils.go

```diff
@@ -521,11 +521,11 @@ func vfIsReady(pciAddr string) (netlink.Link, error) {
 	return vfLink, nil
 }
 
-func setVfsAdminMac(iface *sriovnetworkv1.InterfaceExt) error {
-	glog.Infof("setVfsAdminMac(): device %s", iface.PciAddress)
+func SetVfsAdminMac(iface *sriovnetworkv1.Interface) error {
```

Comment:
is there a reason `InterfaceExt` was originally used here?

Reply:
Not really, both types are ok